### DISCUSSION TITLE
Move lines via dialog

### DISCUSSION THREAD

**Discussion by latvia234** [vincode-io/Zavala#115] [Ideas]
https://user-images.githubusercontent.com/86680397/129060602-b04cd7a0-e708-42e4-bd0e-eb7631865e0a.mp4

I would love it to move lines in Zavala via a dialog. You can open this dialog with a keyboard shortcut. (cmd+shift+m) You can move lines in the outline that is already open, or you can move lines into another outline.

If you have a list with authors, and you are currently reading a book it is under Doing. When you finished reading the book you can open the dialog, search for the author and move the line.

This is just one example but it makes moving lines faster. And a lot faster when you can open the dialog with a keyboard shortcut.

**Comment by vincode-io** [comment]
That looks pretty interesting. I've added it as a Github Issue so that we can include it in a future release.

**Comment by latvia234** [comment]
https://github.com/vincode-io/Zavala/issues/116
